includes/wikia/services/sass/Filter/CdnRewriteFilter.class.php

```diff
@@ -20,14 +20,14 @@ public function __construct( $cdnUrl ) {
 	public function process( $contents ) {
 		wfProfileIn(__METHOD__);
 
-		if (strpos($contents, "wgCdnStylePath") !== false) { // faster to skip the regex in most cases
+		if (strpos($contents, "wgResourceBasePath") !== false) { // faster to skip the regex in most cases
 			// Because of fonts in CSS, we have to allow for lines with multiple url()s in them.
 			// This will rewrite all but the last URL on the line (the last regex will fix the final URL and remove the special comment).
 			$wasChanged = true;
 
 			// TODO: refactor?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app